### DISCUSSION TITLE
Aide pour avoir les chaînes françaises

### DISCUSSION THREAD

**Discussion by G52-hadi** [iptv-org/community#1096] [Q&A]
Salut s'il vous plaît je besoin d'aide pour avoir les chaînes françaises en Afrique merci

**Comment by DarckAngel242** [comment]
Salut ! Avais-tu pu obtenu les chaînes françaises ?

**Comment by freearhey** [comment]
https://iptv-org.github.io/iptv/languages/fra.m3u
